app/src/main/java/com/checkin/app/checkin/restaurant/activities/PublicRestaurantProfileActivity.kt

```diff
@@ -393,6 +393,22 @@ class PublicRestaurantProfileActivity : BaseActivity(), AppBarLayout.OnOffsetCha
             } else "-"
         } ?: "-"
         tvRating.text = restaurantModel.formatRating()
+          val ratingtext: String = tvRating.text.toString();
```

Comment:
Why don't you use `restaurantModel.rating` on RHS of assignment? Then you don't need to cover it back to float.